t/t1011-job-archive-interface.t

```diff
@@ -0,0 +1,95 @@
+#!/bin/bash
+
+test_description='Tests for update-usage command'
+
+. $(dirname $0)/sharness.sh
+
+DB_PATH=$(pwd)/FluxAccountingTest.db
+ARCHIVEDIR=`pwd`
+ARCHIVEDB="${ARCHIVEDIR}/jobarchive.db"
+QUERYCMD="flux python ${SHARNESS_TEST_SRCDIR}/scripts/query.py"
+
+export FLUX_CONF_DIR=$(pwd)
+test_under_flux 4 job
+
+wait_db() {
```

Comment:
might be worthwhile to add a small comment on what this function does, took a little time to grok

Reply:
That function was lifted from flux-core job-archive tests. Maybe we could add the comment there and then copy that to this PR verbatim?

Reply:
oh haha, i hadn't even realized I wrote it :P

Reply:
just pushed PR in flux-core, @cmoussa1, you can copy and paste comments from there.  https://github.com/flux-framework/flux-core/pull/4167

Reply:
Great, thanks! I'll copy and paste here and force-push a fix.

Reply:
Just force-pushed a fix to [b8ac7d3](https://github.com/flux-framework/flux-accounting/pull/211/commits/b8ac7d313209b7326be1421de8b02c09647baf83) to add a function description that matches the one you wrote for flux-framework/flux-core#4167.